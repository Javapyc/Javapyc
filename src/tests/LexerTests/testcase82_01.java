// test1

/* test2 */

///* test3

shouldappear1

/* // test4a
test4b
*/

shouldappear2

/*/ test5 */

/ * shouldappear3 */ 

/*/*/

/**/

/***/

/*////*/

/*//**/

/*//***/

*/

/*/*/*test5*/

shouldappear4

// test6*/

shouldappear5//test7
shouldappear6/*test8*/
