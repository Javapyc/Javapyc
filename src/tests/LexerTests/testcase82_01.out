ID, shouldappear1
ID, shouldappear2
Operator, /
Operator, *
ID, shouldappear3
Operator, *
Operator, /
Operator, *
Operator, /
Operator, *
ID, test5
Operator, *
Operator, /
ID, shouldappear4
ID, shouldappear5
ID, shouldappear6
